.storybook/stories/datagrid/datagrid-row.stories.ts

```diff
@@ -58,12 +58,18 @@ const RowTemplate: StoryFn = args => ({
         (clrDgExpandedChange)="index === 0 && clrDgExpandedChange($event)"
         (clrDgSelectedChange)="index === 0 && clrDgSelectedChange($event)"
       >
-        <clr-dg-cell>{{ element.name }}</clr-dg-cell>
-        <clr-dg-cell>{{ element.symbol }}</clr-dg-cell>
-        <clr-dg-cell>{{ element.number }}</clr-dg-cell>
+        <clr-dg-cell>{{ !(emptyRow && index === 0) ? element.name : '' }}</clr-dg-cell>
```

Comment:
```suggestion
        <clr-dg-cell>{{ emptyRow && index === 0 ? '' : element.name }}</clr-dg-cell>
```

I'm sorry I didn't catch this in my previous review, but conditionals are easier to read without unnecessary negation. I think this should be simplified.

Reply:
Corrected in commit a9a179d